server/views/components/series-container/index.njk

```diff
@@ -1,6 +1,6 @@
 <div class="row">
   <div class="container">
-    <div class="grid">
+    <div class="grid grid--dividers">
```

Comment:
I prefer composing what my content will look like in the HTML by adding CSS classes.

Creating a coupling of css style => css component => html component leaves a lot more room to break things.

From my experience in the past, especially with large sites, being stupid with CSS is a good rule (that's basically what BEM is).